### PR TITLE
admin side page build

### PR DESCRIPTION
**Pull request by jatinpknowarth**

### CODE REVIEW

File: app/Http/Controllers/admin/ProfileController.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+use App\Http\Controllers\Controller;
+
+class ProfileController extends Controller
+{
+	public function __construct()
+    {
+        $this->middleware('auth');
+    }
+    
+    public function editprofile_admin()
```
Comment (kandarp26) [inline app/Http/Controllers/admin/ProfileController.php]: Function name should be edit only. editprofile_admin >> edit

File: app/Http/Controllers/admin/QuestionController.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        return view('')->with('result',$result);
+    }
+
+    public function create()
+    {
+    	 $result = exam::get();
+        return view('admin.add_question')->with('result',$result); 
+    }
+
+    public function createQuestion($id)
```
Comment (kandarp26) [inline app/Http/Controllers/admin/QuestionController.php]: What is different between create and createQuestion. remove one of method if it is not required.

File: app/Http/routes.php [hunk 3]
```diff
@@ -22,5 +22,27 @@
...
+	Route::get('/', 'HomeController@index');
 	Route::get('/home', 'HomeController@index');
+	Route::get('/admin', 'HomeController@admin');
+	// Starting Route of Admin and User End
+
+	// User Side Routes
+	Route::get('exam','user\ExamController@exam');
+	Route::get('editprofile','user\ProfileController@editprofile_user');
+	Route::get('viewresult','user\ResultController@viewresult');
+
```
Comment (kandarp26) [inline app/Http/routes.php]: Create rout group. All admin side url should be {{base_url}}/admin/{{controller}}/{{action}}

File: app/question.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+class question extends Model
+{
+  protected $table = 'question';
+	protected function validateData($post){
+    	$rules =array('question' => 'required|unique:question',
+                      'opt_a' =>'required',
+                      'opt_b' =>'required',
+                      'opt_c' =>'required',
+                      'opt_d' =>'required',
```
Comment (kandarp26) [inline app/question.php:30]: Why opt_d declared  2 time.

File: resources/views/admin/add_question.blade.php [hunk 5]
```diff
@@ -0,0 +1,56 @@
+@extends('layouts.app')
```
Comment (kandarp26) [inline resources/views/admin/add_question.blade.php:1]: Create proper folder structure . 
admin
        question
             -add
             -edit
             -list

File: app/Http/Controllers/admin/ProfileController.php (hunk 1 above)
Comment (jatinpknowarth) [inline app/Http/Controllers/admin/ProfileController.php]: done..

File: app/Http/Controllers/admin/QuestionController.php (hunk 2 above)
Comment (jatinpknowarth) [inline app/Http/Controllers/admin/QuestionController.php]: we need both create methods for 2 diff. pages..

File: app/question.php (hunk 4 above)
Comment (jatinpknowarth) [inline app/question.php:30]: solved..

**Pull request merged**